docs/source/performance/apq.md

```diff
@@ -76,15 +76,18 @@ ecf4edb46db40b5132295c0291d62fb65d6759a9eedfa4d5d612dd5ec54a6b38
 1. Attempt to execute this query on your running server by providing its hash in a `curl` command, like so:
 
     ```shell
-    curl -g 'http://localhost:4000/graphql?extensions={"persistedQuery":{"version":1,"sha256Hash":"ecf4edb46db40b5132295c0291d62fb65d6759a9eedfa4d5d612dd5ec54a6b38"}}'
+    curl -g http://localhost:4000/graphql \
```

Comment:
Am I using a different version of curl from you or something, or do you need to add a `-G` here (and remove the old `-g` since it isn't needed any more)? Otherwise this sends a POST with `content-type: application/x-www-form-urlencoded` and Apollo Server only wants to parse JSON bodies, not that type.

(Another alternative would be to write JSON here though then you will have to include the content-type header explicitly.  Heh, apparently [curl is growing a new `--json` option that handles this as of this month](https://github.com/curl/curl/commit/32160cae847b4e42d52edb033615f88b12ce5c05) but it'll be a while before docs can expect people to have it.)